OptimoveSDK/Sources/Classes/Optimobile/InApp/InAppManager.swift

```diff
@@ -86,10 +88,7 @@ internal class InAppManager {
         #if DEBUG
         sync(onComplete)
         #else
-        let lastSyncTime = UserDefaults.standard.object(forKey: OptimobileUserDefaultsKey.MESSAGES_LAST_SYNC_TIME.rawValue) as? Date
-        if lastSyncTime != nil && lastSyncTime!.timeIntervalSinceNow < -3600 as Double {
-            sync(onComplete)
-        }
+        syncDebounced(onComplete)
```

Comment:
Note for testing of this, you have to run a release build or flip the conditional above